### PR TITLE
Help Revisions for SAM 2023.12.17 Revision 2

### PR DESCRIPTION
**Pull request by cpaulgilman**
== added after review ==

Marine Energy Use Array Scaling capital cost option

Add trademark symbol to first mention of SAM and System Advisor Model

Remove blank line before Microinverters section of Inverter topic.

Remove reference to net captial cost in description of ITC calculation for commercial financial model.

Fix typo mentioning IBI reduction of size of debt in section about CBI.

== original pr ==

PV uncertainty check boxes for enabling sources.

Fix IBI and CBI depreciation basis reduction descriptions.

Revise descriptions of depreciation basis: Installed cost plus financing costs.

Refine bifacial losses and spatial albedo.

Document system availability and AC losses for PV+Battery affect dispatch, but for generic system and battwatts do not.

Sort out "levelized" output variables. Note that LCOG for return on equity may cause confusion given new LCOS.

Tower and Receiver design values: Runs model at design point with made-up flux map (sine wave) trying to get at receiver thermal power with some assumption of irregular flux. Also update Help.

MSPT receiver page revisions.

Revise IEC 61853 topic to explain temperature, spectral, and optical models.

Add note for CEC model to explain air mass model.

Revise PV tracking option descriptions to clarify how tilt and azimuth inputs work for different options.

Improve descriptions of PV output variables for irradiance.

Fix "with out" typo in Indifference Point topic.

Marine Energy refer to DOE RMP per https://github.com/NREL/SAM/issues/1679

Electricity rates net energy metering with $ credits true up period not optional.

Revised wind losses descriptions for clarity.

Fix retail price dispatch expandable section.

# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] This change modifies variables in existing compute modules. Please see [Checking for PySAM Incompatible API Changes](https://github.com/NREL/SAM/wiki/Checking-for-PySAM-Incompatible-API-Changes).

## Checklist:

If you have added a new compute module in a SSC pull request related to this one, be sure to check the [Process Requirements](https://github.com/NREL/SAM/wiki/Compute-modules-in-SAM).

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: deploy/runtime/help/html/incentives.html [hunk 1]
```diff
@@ -537,7 +537,7 @@ <h3 class="p_Heading3_atocs_" style="page-break-inside: avoid; page-break-after:
...
 <p class="p_VariableDescription"><span class="f_VariableDescription">Available for non-residential projects when one of the depreciation options is active on the <a href="fin_residential.html" class="topiclink">Depreciation</a> page. </span></p>
-<p class="p_VariableDescription"><span class="f_VariableDescription">The check boxes determine whether the basis used to calculate federal depreciation, state depreciation, or both should be reduced by the tax credit amount. When you check the box for an ITC, SAM reduces the depreciable basis by 50% of the ITC amount as specified by U.S. IRS rules.</span></p>
+<p class="p_VariableDescription"><span class="f_VariableDescription">The check boxes determine whether the basis used to calculate federal depreciation, state depreciation, or both should be reduced by the tax credit amount. When you check the box for an ITC, SAM reduces the depreciable basis by 50% of the ITC amount as specified by U.S. Internal Revenue Service rules.</span></p>
 <h4 class="p_Heading4"><span class="f_Heading4">Specifying Annual ITC Values</span></h4>
```
Comment (sjanzou) [inline deploy/runtime/help/html/incentives.html:541]: seems like this is correct for the patch and will need to be revised for the reserve accounts in the fall release

**Review by sjanzou — APPROVED** [review]
Looks good as far as I can tell for the things I worked on!

Also, I committed the SAM api and export config updates from the Windows build

**Comment by brtietz** [comment]
I think this line in the incentives page is incorrect:

image

If one adds a 30% IBI to the PVWatts-commercial default costs, the ITC (correctly) remains at $284,459.03.

I don't see any lines in cashloan.cpp that differentiate residential vs commercial for ITC basis: https://github.com/NREL/ssc/blob/724d05aefd7267c66ba8d926bebd8805201ff797/ssc/cmod_cashloan.cpp#L682

I recommend updating the text to "For the residential and commercial models, SAM applies the ITC percentage to the total installed cost"

**Comment by brtietz** [comment]
Possible typo on the incentives page. This section is talking about CBI, then switches to IBI. Is this intentional?
image

**Comment by brtietz** [comment]
Other than my previous two comments, this looks good. Thanks for updating!

**Review by mjprilliman — APPROVED** [review]
image
Trademarks look good. 

image
Marine looks good, thanks for adding this!

The other 'after review' changes look good to me.

**Comment by brtietz** [comment]
With apologies for my initial review not being sufficiently thorough, there are still a few references to "net capital cost" that are no longer correct.

ITC percentage description:
image


Depreciation page's ITC allocation text:
image

Did we have an alternative phrase that we can substitute in here? "Unallocated basis"?

**Review by brtietz — APPROVED** [review]

**Pull request merged**